{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for lasso",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for lasso fixes the following issues:\n\n- CVE-2021-28091: Fixed XML signature wrapping vulnerability when parsing SAML responses (boo#1186768)\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "openSUSE-2021-1057",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/opensuse-su-2021_1057-1.json"
			},
			{
				"category": "self",
				"summary": "URL for openSUSE-SU-2021:1057-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/NVAZRUCJGDCHRY6W5ARRO5NRB27JQCR6/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for openSUSE-SU-2021:1057-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/NVAZRUCJGDCHRY6W5ARRO5NRB27JQCR6/"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1186768",
				"url": "https://bugzilla.suse.com/1186768"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2021-28091 page",
				"url": "https://www.suse.com/security/cve/CVE-2021-28091/"
			}
		],
		"title": "Security update for lasso",
		"tracking": {
			"current_release_date": "2021-07-19T21:03:40Z",
			"generator": {
				"date": "2021-07-19T21:03:40Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "openSUSE-SU-2021:1057-1",
			"initial_release_date": "2021-07-19T21:03:40Z",
			"revision_history": [
				{
					"date": "2021-07-19T21:03:40Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
								"product": {
									"name": "liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
									"product_id": "liblasso-devel-2.6.1-lp152.2.3.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "liblasso3-2.6.1-lp152.2.3.1.x86_64",
								"product": {
									"name": "liblasso3-2.6.1-lp152.2.3.1.x86_64",
									"product_id": "liblasso3-2.6.1-lp152.2.3.1.x86_64"
								}
							},
							{
								"category": "product_version",
								"name": "python3-lasso-2.6.1-lp152.2.3.1.x86_64",
								"product": {
									"name": "python3-lasso-2.6.1-lp152.2.3.1.x86_64",
									"product_id": "python3-lasso-2.6.1-lp152.2.3.1.x86_64"
								}
							}
						],
						"category": "architecture",
						"name": "x86_64"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "openSUSE Leap 15.2",
								"product": {
									"name": "openSUSE Leap 15.2",
									"product_id": "openSUSE Leap 15.2",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:leap:15.2"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "liblasso-devel-2.6.1-lp152.2.3.1.x86_64 as component of openSUSE Leap 15.2",
					"product_id": "openSUSE Leap 15.2:liblasso-devel-2.6.1-lp152.2.3.1.x86_64"
				},
				"product_reference": "liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
				"relates_to_product_reference": "openSUSE Leap 15.2"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "liblasso3-2.6.1-lp152.2.3.1.x86_64 as component of openSUSE Leap 15.2",
					"product_id": "openSUSE Leap 15.2:liblasso3-2.6.1-lp152.2.3.1.x86_64"
				},
				"product_reference": "liblasso3-2.6.1-lp152.2.3.1.x86_64",
				"relates_to_product_reference": "openSUSE Leap 15.2"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-lasso-2.6.1-lp152.2.3.1.x86_64 as component of openSUSE Leap 15.2",
					"product_id": "openSUSE Leap 15.2:python3-lasso-2.6.1-lp152.2.3.1.x86_64"
				},
				"product_reference": "python3-lasso-2.6.1-lp152.2.3.1.x86_64",
				"relates_to_product_reference": "openSUSE Leap 15.2"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2021-28091",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2021-28091"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "Lasso all versions prior to 2.7.0 has improper verification of a cryptographic signature.",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"openSUSE Leap 15.2:liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
					"openSUSE Leap 15.2:liblasso3-2.6.1-lp152.2.3.1.x86_64",
					"openSUSE Leap 15.2:python3-lasso-2.6.1-lp152.2.3.1.x86_64"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2021-28091",
					"url": "https://www.suse.com/security/cve/CVE-2021-28091"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1186768 for CVE-2021-28091",
					"url": "https://bugzilla.suse.com/1186768"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"openSUSE Leap 15.2:liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
						"openSUSE Leap 15.2:liblasso3-2.6.1-lp152.2.3.1.x86_64",
						"openSUSE Leap 15.2:python3-lasso-2.6.1-lp152.2.3.1.x86_64"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 8.8,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
						"version": "3.1"
					},
					"products": [
						"openSUSE Leap 15.2:liblasso-devel-2.6.1-lp152.2.3.1.x86_64",
						"openSUSE Leap 15.2:liblasso3-2.6.1-lp152.2.3.1.x86_64",
						"openSUSE Leap 15.2:python3-lasso-2.6.1-lp152.2.3.1.x86_64"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2021-07-19T21:03:40Z",
					"details": "important"
				}
			],
			"title": "CVE-2021-28091"
		}
	]
}
